## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 490605`
Commit: `9b4b70a7954e132e64a755a9a08e6ebe29cd978d`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>netbox_4_2</li>
    <li>netbox_4_3</li>
    <li>picosnitch</li>
    <li>picosnitch.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 92 packages failed to build:</summary>
  <ul>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.executorch</li>
    <li>python313Packages.executorch.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-classic</li>
    <li>python313Packages.langchain-classic.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.odc-geo</li>
    <li>python313Packages.odc-geo.dist</li>
    <li>python313Packages.odc-loader</li>
    <li>python313Packages.odc-loader.dist</li>
    <li>python313Packages.odc-stac</li>
    <li>python313Packages.odc-stac.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.torchtune</li>
    <li>python313Packages.torchtune.dist</li>
    <li>python314Packages.databricks-sdk</li>
    <li>python314Packages.databricks-sdk.dist</li>
    <li>python314Packages.langchain</li>
    <li>python314Packages.langchain-anthropic</li>
    <li>python314Packages.langchain-anthropic.dist</li>
    <li>python314Packages.langchain-azure-dynamic-sessions</li>
    <li>python314Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python314Packages.langchain-classic</li>
    <li>python314Packages.langchain-classic.dist</li>
    <li>python314Packages.langchain-community</li>
    <li>python314Packages.langchain-community.dist</li>
    <li>python314Packages.langchain-deepseek</li>
    <li>python314Packages.langchain-deepseek.dist</li>
    <li>python314Packages.langchain-experimental</li>
    <li>python314Packages.langchain-experimental.dist</li>
    <li>python314Packages.langchain-mongodb</li>
    <li>python314Packages.langchain-mongodb.dist</li>
    <li>python314Packages.langchain-openai</li>
    <li>python314Packages.langchain-openai.dist</li>
    <li>python314Packages.langchain-xai</li>
    <li>python314Packages.langchain-xai.dist</li>
    <li>python314Packages.langchain.dist</li>
    <li>python314Packages.langgraph-checkpoint-mongodb</li>
    <li>python314Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python314Packages.langgraph-store-mongodb</li>
    <li>python314Packages.langgraph-store-mongodb.dist</li>
    <li>python314Packages.mlflow</li>
    <li>python314Packages.mlflow.dist</li>
    <li>python314Packages.odc-geo</li>
    <li>python314Packages.odc-geo.dist</li>
    <li>python314Packages.odc-loader</li>
    <li>python314Packages.odc-loader.dist</li>
    <li>python314Packages.odc-stac</li>
    <li>python314Packages.odc-stac.dist</li>
    <li>python314Packages.reflex</li>
    <li>python314Packages.reflex-chakra</li>
    <li>python314Packages.reflex-chakra.dist</li>
    <li>python314Packages.reflex.dist</li>
    <li>python314Packages.sagemaker-mlflow</li>
    <li>python314Packages.sagemaker-mlflow.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 726 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>alpaca</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>authentik</li>
    <li>azure-cli-extensions.rdbms-connect</li>
    <li>azure-cli-extensions.rdbms-connect.dist</li>
    <li>cgt-calc</li>
    <li>cgt-calc.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>contact</li>
    <li>contact.dist</li>
    <li>crewai (python313Packages.crewai)</li>
    <li>crewai.dist (python313Packages.crewai.dist)</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>flattenReferencesGraph</li>
    <li>flattenReferencesGraph.dist</li>
    <li>froide</li>
    <li>froide-govplan</li>
    <li>froide-govplan.dist</li>
    <li>froide.dist</li>
    <li>glitchtip</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>hp2p</li>
    <li>jadx</li>
    <li>langgraph-cli (python313Packages.langgraph-cli)</li>
    <li>langgraph-cli.dist (python313Packages.langgraph-cli.dist)</li>
    <li>lasuite-docs</li>
    <li>lasuite-docs.dist</li>
    <li>lasuite-meet</li>
    <li>lasuite-meet.dist</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>lutris-unwrapped</li>
    <li>marimo (python313Packages.marimo)</li>
    <li>marimo.dist (python313Packages.marimo.dist)</li>
    <li>meshtastic</li>
    <li>meshtastic.dist</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>netbox (netbox_4_4)</li>
    <li>newelle</li>
    <li>nominatim</li>
    <li>nominatim.dist</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>paperless-ngx</li>
    <li>paperless-ngx.dist</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>pgcli (python313Packages.pgcli)</li>
    <li>pgcli.dist (python313Packages.pgcli.dist)</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>py-wacz (python313Packages.wacz)</li>
    <li>py-wacz.dist (python313Packages.wacz.dist)</li>
    <li>python313Packages.aiosql</li>
    <li>python313Packages.aiosql.dist</li>
    <li>python313Packages.aiosql.doc</li>
    <li>python313Packages.altair</li>
    <li>python313Packages.altair.dist</li>
    <li>python313Packages.arviz</li>
    <li>python313Packages.arviz-plots</li>
    <li>python313Packages.arviz-plots.dist</li>
    <li>python313Packages.arviz.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python313Packages.backtesting</li>
    <li>python313Packages.backtesting.dist</li>
    <li>python313Packages.bambi</li>
    <li>python313Packages.bambi.dist</li>
    <li>python313Packages.bokeh</li>
    <li>python313Packages.bokeh.dist</li>
    <li>python313Packages.bsuite</li>
    <li>python313Packages.bsuite.dist</li>
    <li>python313Packages.bumps</li>
    <li>python313Packages.bumps.dist</li>
    <li>python313Packages.chart-studio</li>
    <li>python313Packages.chart-studio.dist</li>
    <li>python313Packages.cltk</li>
    <li>python313Packages.cltk.dist</li>
    <li>python313Packages.clu</li>
    <li>python313Packages.clu.dist</li>
    <li>python313Packages.coiled</li>
    <li>python313Packages.coiled.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.corner</li>
    <li>python313Packages.corner.dist</li>
    <li>python313Packages.dalle-mini</li>
    <li>python313Packages.dalle-mini.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.django-pgactivity</li>
    <li>python313Packages.django-pgactivity.dist</li>
    <li>python313Packages.django-pglock</li>
    <li>python313Packages.django-pglock.dist</li>
    <li>python313Packages.django-postgres-partition</li>
    <li>python313Packages.django-postgres-partition.dist</li>
    <li>python313Packages.django-tenants</li>
    <li>python313Packages.django-tenants.dist</li>
    <li>python313Packages.dm-sonnet</li>
    <li>python313Packages.dm-sonnet.dist</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.energyflow.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.extra-streamlit-components</li>
    <li>python313Packages.extra-streamlit-components.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.folium</li>
    <li>python313Packages.folium.dist</li>
    <li>python313Packages.formulaic</li>
    <li>python313Packages.formulaic.dist</li>
    <li>python313Packages.frictionless</li>
    <li>python313Packages.frictionless.dist</li>
    <li>python313Packages.geoarrow-pyarrow</li>
    <li>python313Packages.geoarrow-pyarrow.dist</li>
    <li>python313Packages.geodatasets</li>
    <li>python313Packages.geodatasets.dist</li>
    <li>python313Packages.geopandas</li>
    <li>python313Packages.geopandas.dist</li>
    <li>python313Packages.geoparquet</li>
    <li>python313Packages.geoparquet.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.harlequin-postgres</li>
    <li>python313Packages.harlequin-postgres.dist</li>
    <li>python313Packages.heudiconv</li>
    <li>python313Packages.heudiconv.dist</li>
    <li>python313Packages.holistic-trace-analysis</li>
    <li>python313Packages.holistic-trace-analysis.dist</li>
    <li>python313Packages.holoviews</li>
    <li>python313Packages.holoviews.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.inequality</li>
    <li>python313Packages.inequality.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.kokoro</li>
    <li>python313Packages.kokoro.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-runtime-inmem</li>
    <li>python313Packages.langgraph-runtime-inmem.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.layoutparser</li>
    <li>python313Packages.layoutparser.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.libpysal</li>
    <li>python313Packages.libpysal.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.lifelines</li>
    <li>python313Packages.lifelines.dist</li>
    <li>python313Packages.linearmodels</li>
    <li>python313Packages.linearmodels.dist</li>
    <li>python313Packages.livelossplot</li>
    <li>python313Packages.livelossplot.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.mapclassify</li>
    <li>python313Packages.mapclassify.dist</li>
    <li>python313Packages.mayim</li>
    <li>python313Packages.mayim.dist</li>
    <li>python313Packages.meshtastic</li>
    <li>python313Packages.meshtastic.dist</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mlcroissant.dist</li>
    <li>python313Packages.moddb</li>
    <li>python313Packages.moddb.dist</li>
    <li>python313Packages.momepy</li>
    <li>python313Packages.momepy.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.narwhals</li>
    <li>python313Packages.narwhals.dist</li>
    <li>python313Packages.nbsmoke</li>
    <li>python313Packages.nbsmoke.dist</li>
    <li>python313Packages.netbox-attachments</li>
    <li>python313Packages.netbox-attachments.dist</li>
    <li>python313Packages.netbox-bgp</li>
    <li>python313Packages.netbox-bgp.dist</li>
    <li>python313Packages.netbox-contract</li>
    <li>python313Packages.netbox-contract.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.netbox-floorplan-plugin</li>
    <li>python313Packages.netbox-floorplan-plugin.dist</li>
    <li>python313Packages.netbox-interface-synchronization</li>
    <li>python313Packages.netbox-interface-synchronization.dist</li>
    <li>python313Packages.netbox-napalm-plugin</li>
    <li>python313Packages.netbox-napalm-plugin.dist</li>
    <li>python313Packages.netbox-plugin-prometheus-sd</li>
    <li>python313Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python313Packages.netbox-qrcode</li>
    <li>python313Packages.netbox-qrcode.dist</li>
    <li>python313Packages.netbox-reorder-rack</li>
    <li>python313Packages.netbox-reorder-rack.dist</li>
    <li>python313Packages.netbox-routing</li>
    <li>python313Packages.netbox-routing.dist</li>
    <li>python313Packages.netbox-topology-views</li>
    <li>python313Packages.netbox-topology-views.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.nipype</li>
    <li>python313Packages.nipype.dist</li>
    <li>python313Packages.niworkflows</li>
    <li>python313Packages.niworkflows.dist</li>
    <li>python313Packages.nominatim-api</li>
    <li>python313Packages.nominatim-api.dist</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.nutpie.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.osmnx</li>
    <li>python313Packages.osmnx.dist</li>
    <li>python313Packages.pandantic</li>
    <li>python313Packages.pandantic.dist</li>
    <li>python313Packages.pandas-stubs</li>
    <li>python313Packages.pandas-stubs.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.panel</li>
    <li>python313Packages.panel.dist</li>
    <li>python313Packages.pdfplumber</li>
    <li>python313Packages.pdfplumber.dist</li>
    <li>python313Packages.pgspecial</li>
    <li>python313Packages.pgspecial.dist</li>
    <li>python313Packages.pgvector</li>
    <li>python313Packages.pgvector.dist</li>
    <li>python313Packages.plopp</li>
    <li>python313Packages.plopp.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.plotnine</li>
    <li>python313Packages.plotnine.dist</li>
    <li>python313Packages.pproxy</li>
    <li>python313Packages.pproxy.dist</li>
    <li>python313Packages.psnawp</li>
    <li>python313Packages.psnawp.dist</li>
    <li>python313Packages.psycopg</li>
    <li>python313Packages.psycopg.dist</li>
    <li>python313Packages.psycopg.doc</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.pybids</li>
    <li>python313Packages.pybids.dist</li>
    <li>python313Packages.pymc</li>
    <li>python313Packages.pymc.dist</li>
    <li>python313Packages.pymoo</li>
    <li>python313Packages.pymoo.dist</li>
    <li>python313Packages.pyrate-limiter</li>
    <li>python313Packages.pyrate-limiter.dist</li>
    <li>python313Packages.pyreadstat</li>
    <li>python313Packages.pyreadstat.dist</li>
    <li>python313Packages.pytest-postgresql</li>
    <li>python313Packages.pytest-postgresql.dist</li>
    <li>python313Packages.python-roborock</li>
    <li>python313Packages.python-roborock.dist</li>
    <li>python313Packages.requests-hardened</li>
    <li>python313Packages.requests-hardened.dist</li>
    <li>python313Packages.sasmodels</li>
    <li>python313Packages.sasmodels.dist</li>
    <li>python313Packages.scikit-tda</li>
    <li>python313Packages.scikit-tda.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.skops</li>
    <li>python313Packages.skops.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.skytemple-dtef</li>
    <li>python313Packages.skytemple-dtef.dist</li>
    <li>python313Packages.skytemple-files</li>
    <li>python313Packages.skytemple-files.dist</li>
    <li>python313Packages.skytemple-ssb-debugger</li>
    <li>python313Packages.skytemple-ssb-debugger.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.sqlframe</li>
    <li>python313Packages.sqlframe.dist</li>
    <li>python313Packages.st-pages</li>
    <li>python313Packages.st-pages.dist</li>
    <li>streamlit (python313Packages.streamlit)</li>
    <li>python313Packages.streamlit-avatar</li>
    <li>python313Packages.streamlit-avatar.dist</li>
    <li>python313Packages.streamlit-card</li>
    <li>python313Packages.streamlit-card.dist</li>
    <li>python313Packages.streamlit-echarts</li>
    <li>python313Packages.streamlit-echarts.dist</li>
    <li>python313Packages.streamlit-folium</li>
    <li>python313Packages.streamlit-folium.dist</li>
    <li>python313Packages.streamlit-kpi-card</li>
    <li>python313Packages.streamlit-kpi-card.dist</li>
    <li>python313Packages.streamlit-notify</li>
    <li>python313Packages.streamlit-notify.dist</li>
    <li>streamlit.dist (python313Packages.streamlit.dist)</li>
    <li>python313Packages.svgdigitizer</li>
    <li>python313Packages.svgdigitizer.dist</li>
    <li>python313Packages.swh-auth</li>
    <li>python313Packages.swh-auth.dist</li>
    <li>python313Packages.swh-core</li>
    <li>python313Packages.swh-core.dist</li>
    <li>python313Packages.swh-export</li>
    <li>python313Packages.swh-export.dist</li>
    <li>python313Packages.swh-journal</li>
    <li>python313Packages.swh-journal.dist</li>
    <li>python313Packages.swh-objstorage</li>
    <li>python313Packages.swh-objstorage.dist</li>
    <li>python313Packages.swh-scanner</li>
    <li>python313Packages.swh-scanner.dist</li>
    <li>python313Packages.swh-scheduler</li>
    <li>python313Packages.swh-scheduler.dist</li>
    <li>python313Packages.swh-storage</li>
    <li>python313Packages.swh-storage.dist</li>
    <li>python313Packages.swh-web-client</li>
    <li>python313Packages.swh-web-client.dist</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.templateflow</li>
    <li>python313Packages.templateflow.dist</li>
    <li>python313Packages.tensorflow-datasets</li>
    <li>python313Packages.tensorflow-datasets.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>python313Packages.vega</li>
    <li>python313Packages.vega.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>python313Packages.wktutils</li>
    <li>python313Packages.wktutils.dist</li>
    <li>python314Packages.aiosql</li>
    <li>python314Packages.aiosql.dist</li>
    <li>python314Packages.aiosql.doc</li>
    <li>python314Packages.altair</li>
    <li>python314Packages.altair.dist</li>
    <li>python314Packages.arviz-plots</li>
    <li>python314Packages.arviz-plots.dist</li>
    <li>python314Packages.backtesting</li>
    <li>python314Packages.backtesting.dist</li>
    <li>python314Packages.bokeh</li>
    <li>python314Packages.bokeh.dist</li>
    <li>python314Packages.bsuite</li>
    <li>python314Packages.bsuite.dist</li>
    <li>python314Packages.bumps</li>
    <li>python314Packages.bumps.dist</li>
    <li>python314Packages.chart-studio</li>
    <li>python314Packages.chart-studio.dist</li>
    <li>python314Packages.coiled</li>
    <li>python314Packages.coiled.dist</li>
    <li>python314Packages.compressai</li>
    <li>python314Packages.compressai.dist</li>
    <li>python314Packages.dash</li>
    <li>python314Packages.dash-bootstrap-components</li>
    <li>python314Packages.dash-bootstrap-components.dist</li>
    <li>python314Packages.dash-bootstrap-templates</li>
    <li>python314Packages.dash-bootstrap-templates.dist</li>
    <li>python314Packages.dash.dist</li>
    <li>python314Packages.django-pgactivity</li>
    <li>python314Packages.django-pgactivity.dist</li>
    <li>python314Packages.django-pglock</li>
    <li>python314Packages.django-pglock.dist</li>
    <li>python314Packages.django-postgres-partition</li>
    <li>python314Packages.django-postgres-partition.dist</li>
    <li>python314Packages.django-tenants</li>
    <li>python314Packages.django-tenants.dist</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.experiment-utilities.dist</li>
    <li>python314Packages.explorerscript</li>
    <li>python314Packages.explorerscript.dist</li>
    <li>python314Packages.extra-streamlit-components</li>
    <li>python314Packages.extra-streamlit-components.dist</li>
    <li>python314Packages.folium</li>
    <li>python314Packages.folium.dist</li>
    <li>python314Packages.formulaic</li>
    <li>python314Packages.formulaic.dist</li>
    <li>python314Packages.geoarrow-pyarrow</li>
    <li>python314Packages.geoarrow-pyarrow.dist</li>
    <li>python314Packages.geodatasets</li>
    <li>python314Packages.geodatasets.dist</li>
    <li>python314Packages.geopandas</li>
    <li>python314Packages.geopandas.dist</li>
    <li>python314Packages.geoparquet</li>
    <li>python314Packages.geoparquet.dist</li>
    <li>python314Packages.gradio</li>
    <li>python314Packages.gradio.dist</li>
    <li>python314Packages.great-tables</li>
    <li>python314Packages.great-tables.dist</li>
    <li>python314Packages.harlequin-postgres</li>
    <li>python314Packages.harlequin-postgres.dist</li>
    <li>python314Packages.heudiconv</li>
    <li>python314Packages.heudiconv.dist</li>
    <li>python314Packages.holistic-trace-analysis</li>
    <li>python314Packages.holistic-trace-analysis.dist</li>
    <li>python314Packages.holoviews</li>
    <li>python314Packages.holoviews.dist</li>
    <li>python314Packages.hvplot</li>
    <li>python314Packages.hvplot.dist</li>
    <li>python314Packages.igraph</li>
    <li>python314Packages.igraph.dist</li>
    <li>python314Packages.inequality</li>
    <li>python314Packages.inequality.dist</li>
    <li>python314Packages.intake</li>
    <li>python314Packages.intake.dist</li>
    <li>python314Packages.iplotx</li>
    <li>python314Packages.iplotx.dist</li>
    <li>python314Packages.itables</li>
    <li>python314Packages.itables.dist</li>
    <li>python314Packages.k-diffusion</li>
    <li>python314Packages.k-diffusion.dist</li>
    <li>python314Packages.kmapper</li>
    <li>python314Packages.kmapper.dist</li>
    <li>python314Packages.langgraph</li>
    <li>python314Packages.langgraph-checkpoint-postgres</li>
    <li>python314Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python314Packages.langgraph-cli</li>
    <li>python314Packages.langgraph-cli.dist</li>
    <li>python314Packages.langgraph-prebuilt</li>
    <li>python314Packages.langgraph-prebuilt.dist</li>
    <li>python314Packages.langgraph-runtime-inmem</li>
    <li>python314Packages.langgraph-runtime-inmem.dist</li>
    <li>python314Packages.langgraph.dist</li>
    <li>python314Packages.layoutparser</li>
    <li>python314Packages.layoutparser.dist</li>
    <li>python314Packages.leidenalg</li>
    <li>python314Packages.leidenalg.dist</li>
    <li>python314Packages.libpysal</li>
    <li>python314Packages.libpysal.dist</li>
    <li>python314Packages.lida</li>
    <li>python314Packages.lida.dist</li>
    <li>python314Packages.lifelines</li>
    <li>python314Packages.lifelines.dist</li>
    <li>python314Packages.linearmodels</li>
    <li>python314Packages.linearmodels.dist</li>
    <li>python314Packages.livelossplot</li>
    <li>python314Packages.livelossplot.dist</li>
    <li>python314Packages.mapclassify</li>
    <li>python314Packages.mapclassify.dist</li>
    <li>python314Packages.marimo</li>
    <li>python314Packages.marimo.dist</li>
    <li>python314Packages.mayim</li>
    <li>python314Packages.mayim.dist</li>
    <li>python314Packages.meshtastic</li>
    <li>python314Packages.meshtastic.dist</li>
    <li>python314Packages.mlcroissant</li>
    <li>python314Packages.mlcroissant.dist</li>
    <li>python314Packages.moddb</li>
    <li>python314Packages.moddb.dist</li>
    <li>python314Packages.momepy</li>
    <li>python314Packages.momepy.dist</li>
    <li>python314Packages.msticpy</li>
    <li>python314Packages.msticpy.dist</li>
    <li>python314Packages.narwhals</li>
    <li>python314Packages.narwhals.dist</li>
    <li>python314Packages.nbsmoke</li>
    <li>python314Packages.nbsmoke.dist</li>
    <li>python314Packages.netbox-bgp</li>
    <li>python314Packages.netbox-bgp.dist</li>
    <li>python314Packages.netbox-documents</li>
    <li>python314Packages.netbox-documents.dist</li>
    <li>python314Packages.netbox-interface-synchronization</li>
    <li>python314Packages.netbox-interface-synchronization.dist</li>
    <li>python314Packages.netbox-plugin-prometheus-sd</li>
    <li>python314Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python314Packages.netbox-qrcode</li>
    <li>python314Packages.netbox-qrcode.dist</li>
    <li>python314Packages.netbox-reorder-rack</li>
    <li>python314Packages.netbox-reorder-rack.dist</li>
    <li>python314Packages.netbox-routing</li>
    <li>python314Packages.netbox-routing.dist</li>
    <li>python314Packages.neurokit2</li>
    <li>python314Packages.neurokit2.dist</li>
    <li>python314Packages.niaarm</li>
    <li>python314Packages.niaarm.dist</li>
    <li>python314Packages.nipype</li>
    <li>python314Packages.nipype.dist</li>
    <li>python314Packages.niworkflows</li>
    <li>python314Packages.niworkflows.dist</li>
    <li>python314Packages.nominatim-api</li>
    <li>python314Packages.nominatim-api.dist</li>
    <li>python314Packages.osmnx</li>
    <li>python314Packages.osmnx.dist</li>
    <li>python314Packages.pandantic</li>
    <li>python314Packages.pandantic.dist</li>
    <li>python314Packages.pandas-stubs</li>
    <li>python314Packages.pandas-stubs.dist</li>
    <li>python314Packages.pandera</li>
    <li>python314Packages.pandera.dist</li>
    <li>python314Packages.panel</li>
    <li>python314Packages.panel.dist</li>
    <li>python314Packages.pdfplumber</li>
    <li>python314Packages.pdfplumber.dist</li>
    <li>python314Packages.pgcli</li>
    <li>python314Packages.pgcli.dist</li>
    <li>python314Packages.pgspecial</li>
    <li>python314Packages.pgspecial.dist</li>
    <li>python314Packages.pgvector</li>
    <li>python314Packages.pgvector.dist</li>
    <li>python314Packages.plopp</li>
    <li>python314Packages.plopp.dist</li>
    <li>python314Packages.plotly</li>
    <li>python314Packages.plotly.dist</li>
    <li>python314Packages.plotnine</li>
    <li>python314Packages.plotnine.dist</li>
    <li>python314Packages.pproxy</li>
    <li>python314Packages.pproxy.dist</li>
    <li>python314Packages.psnawp</li>
    <li>python314Packages.psnawp.dist</li>
    <li>python314Packages.psycopg</li>
    <li>python314Packages.psycopg.dist</li>
    <li>python314Packages.psycopg.doc</li>
    <li>python314Packages.pybids</li>
    <li>python314Packages.pybids.dist</li>
    <li>python314Packages.pyrate-limiter</li>
    <li>python314Packages.pyrate-limiter.dist</li>
    <li>python314Packages.pyreadstat</li>
    <li>python314Packages.pyreadstat.dist</li>
    <li>python314Packages.pytest-postgresql</li>
    <li>python314Packages.pytest-postgresql.dist</li>
    <li>python314Packages.python-roborock</li>
    <li>python314Packages.python-roborock.dist</li>
    <li>python314Packages.requests-hardened</li>
    <li>python314Packages.requests-hardened.dist</li>
    <li>python314Packages.sasmodels</li>
    <li>python314Packages.sasmodels.dist</li>
    <li>python314Packages.scikit-tda</li>
    <li>python314Packages.scikit-tda.dist</li>
    <li>python314Packages.shiny</li>
    <li>python314Packages.shiny.dist</li>
    <li>python314Packages.skops</li>
    <li>python314Packages.skops.dist</li>
    <li>python314Packages.skytemple-dtef</li>
    <li>python314Packages.skytemple-dtef.dist</li>
    <li>python314Packages.skytemple-files</li>
    <li>python314Packages.skytemple-files.dist</li>
    <li>python314Packages.skytemple-ssb-debugger</li>
    <li>python314Packages.skytemple-ssb-debugger.dist</li>
    <li>python314Packages.smolagents</li>
    <li>python314Packages.smolagents.dist</li>
    <li>python314Packages.spacy-loggers</li>
    <li>python314Packages.spacy-loggers.dist</li>
    <li>python314Packages.sqlframe</li>
    <li>python314Packages.sqlframe.dist</li>
    <li>python314Packages.st-pages</li>
    <li>python314Packages.st-pages.dist</li>
    <li>python314Packages.streamlit</li>
    <li>python314Packages.streamlit-avatar</li>
    <li>python314Packages.streamlit-avatar.dist</li>
    <li>python314Packages.streamlit-card</li>
    <li>python314Packages.streamlit-card.dist</li>
    <li>python314Packages.streamlit-echarts</li>
    <li>python314Packages.streamlit-echarts.dist</li>
    <li>python314Packages.streamlit-folium</li>
    <li>python314Packages.streamlit-folium.dist</li>
    <li>python314Packages.streamlit-kpi-card</li>
    <li>python314Packages.streamlit-kpi-card.dist</li>
    <li>python314Packages.streamlit-notify</li>
    <li>python314Packages.streamlit-notify.dist</li>
    <li>python314Packages.streamlit.dist</li>
    <li>python314Packages.synergy</li>
    <li>python314Packages.synergy.dist</li>
    <li>python314Packages.templateflow</li>
    <li>python314Packages.templateflow.dist</li>
    <li>python314Packages.txtai</li>
    <li>python314Packages.txtai.dist</li>
    <li>python314Packages.unstructured-inference</li>
    <li>python314Packages.unstructured-inference.dist</li>
    <li>python314Packages.vega</li>
    <li>python314Packages.vega.dist</li>
    <li>python314Packages.wandb</li>
    <li>python314Packages.wandb.dist</li>
    <li>python314Packages.wktutils</li>
    <li>python314Packages.wktutils.dist</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>sasview</li>
    <li>sasview.dist</li>
    <li>skytemple</li>
    <li>skytemple.dist</li>
    <li>sptk</li>
    <li>supercell-wx</li>
    <li>swh</li>
    <li>tests.home-assistant-component-tests.playstation_network</li>
    <li>tests.home-assistant-component-tests.roborock</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>tocpdf</li>
    <li>tocpdf.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>umap</li>
    <li>umap.dist</li>
    <li>unstructured-api</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python313Packages.langchain</summary>
<pre>tests/unit_tests/agents/test_return_direct_spec.py:35: 12 warnings
  /build/source/libs/langchain_v1/tests/unit_tests/agents/test_return_direct_spec.py:35: PytestCollectionWarning: cannot collect test class &#x27;TestCase&#x27; because it has a __init__ constructor (from: tests/unit_tests/agents/test_return_direct_spec.py)
    class TestCase(BaseSchema):

tests/unit_tests/agents/middleware/implementations/test_summarization.py::test_summarization_middleware_full_workflow
  /build/source/libs/langchain_v1/tests/unit_tests/agents/middleware/implementations/test_summarization.py:428: DeprecationWarning: max_tokens_before_summary is deprecated. Use trigger=(&#x27;tokens&#x27;, value) instead.
    middleware = SummarizationMiddleware(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================= slowest 5 durations ==============================
1.02s call     tests/unit_tests/agents/middleware/implementations/test_shell_tool.py::test_startup_command_failure
1.02s call     tests/unit_tests/agents/middleware/implementations/test_shell_tool.py::test_shutdown_command_failure_logged
0.78s call     tests/unit_tests/agents/middleware/implementations/test_tool_retry.py::test_tool_retry_backoff_timing
0.75s call     tests/unit_tests/agents/middleware/implementations/test_tool_retry.py::test_tool_retry_async_backoff_timing
0.75s call     tests/unit_tests/agents/middleware/implementations/test_model_retry.py::test_model_retry_backoff_timing
=========================== short test summary info ============================
FAILED tests/unit_tests/agents/middleware/implementations/test_shell_tool.py::test_truncation_by_bytes - AssertionError: assert &#x27;truncated at 50 bytes&#x27; in &#x27;[stderr] /nix/store/f15k3dpilmiyv6zgpib289rnjykgr1r4-bash-5.3p9/bin/bash: line 1: python3: command not found\n\nexit code: 127&#x27;
 +  where &#x27;[stderr] /nix/store/f15k3dpilmiyv6zgpib289rnjykgr1r4-bash-5.3p9/bin/bash: line 1: python3: command not found\n\nexit code: 127&#x27; = &lt;built-in method lower of str object at 0x7fffeebdbd60&gt;()
 +    where &lt;built-in method lower of str object at 0x7fffeebdbd60&gt; = &#x27;[stderr] /nix/store/f15k3dpilmiyv6zgpib289rnjykgr1r4-bash-5.3p9/bin/bash: line 1: python3: command not found\n\nExit code: 127&#x27;.lower
=========== 1 failed, 696 passed, 16 skipped, 25 warnings in 21.85s ============</pre>
</details>
<details>
<summary>python313Packages.odc-geo</summary>
<pre>WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in transformer options does not support option DTYPE
WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in transformer options does not support option XSCALE
WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in transformer options does not support option YSCALE
WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in warp options does not support option DTYPE
WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in transformer options does not support option DTYPE
WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in transformer options does not support option XSCALE
WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in transformer options does not support option YSCALE
WARNING  rasterio._env:warp.py:387 CPLE_NotSupported in warp options does not support option DTYPE
______________________ test_xr_reproject[xx_chunks1-None] ______________________

xx_epsg4326 = &lt;xarray.DataArray &#x27;zeros_like-e77fdc5e5040c3a4d88e3149f5e710b4&#x27; (latitude: 4,
                                        ... float64 64B -17.5 -12.5 -7.5 -2.5 2.5 7.5 12.5 17.5
    spatial_ref  int32 4B 4326
Attributes:
    crs:      epsg:4326

    @pytest.mark.parametrize(&quot;xx_time&quot;, [None, [&quot;2020-01-30&quot;]])
    @pytest.mark.parametrize(&quot;xx_chunks&quot;, [None, (-1, -1), (4, 4)])
    def test_xr_reproject(xx_epsg4326: xr.DataArray):
        assert isinstanc</pre>
</details>
<details>
<summary>python313Packages.reflex</summary>
<pre>  /build/source/reflex/utils/types.py:382: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

tests/units/test_attribute_access_type.py: 10 warnings
  /build/source/reflex/utils/types.py:384: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

tests/units/test_model.py::test_upcast_event_handler_arg[handler0-payload0]
  /build/source/reflex/state.py:1912: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/units/test_state.py::test_state_manager_lock_expire - reflex.utils.exceptions.LockExpiredError: Lock expired for token 9dd07dd0-b...
FAILED tests/units/test_state.py::test_state_manager_lock_expire_contend - reflex.utils.exceptions.LockExpiredError: Lock expired for token 7c1c14db-5...
FAILED tests/units/test_state.py::test_state_manager_lock_warning_threshold_contend - reflex.utils.exceptions.LockExpiredError: Lock expired for token 787f921f-a...
= 3 failed, 3505 passed, 17 skipped, 31 deselected, 75 warnings in 112.56s (0:01:52) =</pre>
</details>
<details>
<summary>python314Packages.langchain</summary>
<pre>tests/unit_tests/agents/middleware/implementations/test_summarization.py::test_summarization_middleware_full_workflow
  /build/source/libs/langchain_v1/tests/unit_tests/agents/middleware/implementations/test_summarization.py:428: DeprecationWarning: max_tokens_before_summary is deprecated. Use trigger=(&#x27;tokens&#x27;, value) instead.
    middleware = SummarizationMiddleware(

tests/unit_tests/agents/test_agent_name.py: 18 warnings
  /nix/store/xw6g4q2nxl0chqd8v985rpf6xxaa9rgw-python3.14-langchain-core-1.2.7/lib/python3.14/site-packages/langchain_core/callbacks/manager.py:379: DeprecationWarning: &#x27;asyncio.iscoroutinefunction&#x27; is deprecated and slated for removal in Python 3.16; use inspect.iscoroutinefunction() instead
    if asyncio.iscoroutinefunction(event):

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================= slowest 5 durations ==============================
1.02s call     tests/unit_tests/agents/middleware/implementations/test_shell_tool.py::test_shutdown_command_failure_logged
1.02s call     tests/unit_tests/agents/middleware/implementations/test_shell_tool.py::test_startup_command_failure
0.80s call     tests/unit_tests/agents/middleware/implementations/test_tool_retry.py::test_tool_retry_backoff_timing
0.79s call     tests/unit_tests/agents/middleware/implementations/test_tool_retry.py::test_tool_retry_async_backoff_timing
0.75s call     tests/unit_tests/agents/middleware/implementations/test_model_retry.py::test_model_retry_backoff_timing
=========================== short test summary info ============================
FAILED tests/unit_tests/agents/middleware/implementations/test_shell_tool.py::test_truncation_by_bytes - AssertionError: assert &#x27;truncated at 50 bytes&#x27; in &#x27;[stderr] /nix/store/f15k3dpilmiyv6zgpib289rnjykgr1r4-bash-5.3p9/bin/bash: line 1: python3: command not found\n\nexit code: 127&#x27;
 +  where &#x27;[stderr] /nix/store/f15k3dpilmiyv6zgpib289rnjykgr1r4-bash-5.3p9/bin/bash: line 1: python3: command not found\n\nexit code: 127&#x27; = &lt;built-in method lower of str object at 0x7fffee324ea0&gt;()
 +    where &lt;built-in method lower of str object at 0x7fffee324ea0&gt; = &#x27;[stderr] /nix/store/f15k3dpilmiyv6zgpib289rnjykgr1r4-bash-5.3p9/bin/bash: line 1: python3: command not found\n\nExit code: 127&#x27;.lower
=========== 1 failed, 691 passed, 21 skipped, 56 warnings in 19.92s ============</pre>
</details>
<details>
<summary>python314Packages.odc-geo</summary>
<pre>    
        assert xx_epsg4326.odc.reproject(3857, dtype=&quot;float32&quot;).dtype == &quot;float32&quot;
    
        yy = xr.Dataset({&quot;a&quot;: xx0, &quot;b&quot;: xx0 + 1, &quot;c&quot;: xr.DataArray([2, 3, 4])})
        assert isinstance(yy.odc, ODCExtensionDs)
        assert yy.odc.geobox == xx0.odc.geobox
        yy_ = yy.odc.reproject(dst_gbox)
        assert isinstance(yy_.odc, ODCExtensionDs)
        assert yy_.odc.geobox == dst_gbox
        assert yy_.a.odc.geobox == dst_gbox
        assert yy_.b.odc.geobox == dst_gbox
        assert (yy_.c == yy.c).all()
    
        yy_ = yy.odc.reproject(&quot;utm&quot;)
&gt;       assert yy_.odc.geobox.crs.proj.utm_zone is not None
E       assert None is not None
E        +  where None = &lt;Geographic 2D CRS: EPSG:4326&gt;\nName: WGS 84\nAxis Info [ellipsoidal]:\n- Lat[north]: Geodetic latitude (degree)\n- Lon[ea...-180.0, -90.0, 180.0, 90.0)\nDatum: World Geodetic System 1984 ensemble\n- Ellipsoid: WGS 84\n- Prime Meridian: Greenwich\n.utm_zone
E        +    where &lt;Geographic 2D CRS: EPSG:4326&gt;\nName: WGS 84\nAxis Info [ellipsoidal]:\n- Lat[north]: Geodetic latitude (degree)\n- Lon[ea...-180.0, -90.0, 180.0, 90.0)\nDatum: World Geodetic System 1984 ensemble\n- Ellipsoid: WGS 84\n- Prime Meridian: Greenwich\n = CRS(&#x27;GEOGCRS[&quot;WGS 84&quot;,ENSEMBLE[&quot;World Geodetic System 1984 ensemble&quot;,MEMBER[&quot;World Geodetic System 1984 (Transit)&quot;],ME...2925199433]],USAGE[SCOPE[&quot;Horizontal component of 3D system.&quot;],AREA[&quot;World.&quot;],BBOX[-90,-180,90,180]],ID[&quot;EPSG&quot;,4326]]&#x27;).proj
E        +      where CRS(&#x27;GEOGCRS[&quot;WGS 84&quot;,ENSEMBLE[&quot;World Geodetic System 1984 ensemble&quot;,MEMBER[&quot;World Geodetic System 1984 (Transit)&quot;],ME...2925199433]],USAGE[SCOPE[&quot;Horizontal component of 3D system.&quot;],AREA[&quot;World.&quot;],BBOX[-90,-180,90,180]],ID[&quot;EPSG&quot;,4326]]&#x27;) = GeoBox((8, 11), Affine(566047.0585719852, 0.0, -1698141.1757159554,\n       0.0, -566047.0585719852, 2264188.234287941)...925199433]],USAGE[SCOPE[&quot;Horizontal component of 3D system.&quot;],AREA[&quot;World.&quot;],BBOX[-90,-180,90,180]],ID[&quot;EPSG&quot;,4326]]&#x27;)).crs
E       </pre>
</details>
<details>
<summary>python314Packages.reflex</summary>
<pre>  /build/source/reflex/utils/types.py:382: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

tests/units/test_attribute_access_type.py: 10 warnings
  /build/source/reflex/utils/types.py:384: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

tests/units/test_model.py::test_upcast_event_handler_arg[handler0-payload0]
  /build/source/reflex/state.py:1912: PydanticDeprecatedSince20:
  
  The `__fields__` attribute is deprecated, use the `model_fields` class property instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.12/migration/

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/units/test_state.py::test_state_manager_lock_expire - reflex.utils.exceptions.LockExpiredError: Lock expired for token b3f1cfe7-c...
FAILED tests/units/test_state.py::test_state_manager_lock_expire_contend - reflex.utils.exceptions.LockExpiredError: Lock expired for token 7edc54be-c...
FAILED tests/units/test_state.py::test_state_manager_lock_warning_threshold_contend - reflex.utils.exceptions.LockExpiredError: Lock expired for token 81859583-9...
= 3 failed, 3505 passed, 17 skipped, 31 deselected, 76 warnings in 83.20s (0:01:23) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/pproxy/default.nix
+++ b/pkgs/development/python-modules/pproxy/default.nix
@@ -18,8 +18,8 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "qwj";
     repo = "python-proxy";
-    rev = "7fccf8dd62204f34b0aa3a70fc568fd6ddff7728";
-    sha256 = "sha256-bOqDdNiaZ5MRi/UeF0hJwMs+rfQBKRsTmXrZ6ieIguo=";
+    tag = version;
+    hash = "sha256-DWxbU2LtXzec1T175cMVJuWuhnxWYhe0FH67stMyOTM=";
   };
 
   nativeBuildInputs = [ setuptools ];
@@ -58,6 +58,6 @@ buildPythonPackage rec {
     mainProgram = "pproxy";
     homepage = "https://github.com/qwj/python-proxy";
     license = lib.licenses.mit;
-    maintainers = [ ];
+    maintainers = [ lib.maintainers.ryand56 ];
   };
 }

--- a/pkgs/development/python-modules/requests-hardened/default.nix
+++ b/pkgs/development/python-modules/requests-hardened/default.nix
@@ -1,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  poetry-core,
+  requests,
+
+  pproxy,
+  pytest-socket,
+  pysocks,
+  trustme,
+  pytestCheckHook,
+}:
+buildPythonPackage rec {
+  pname = "requests-hardened";
+  version = "1.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "saleor";
+    repo = "requests-hardened";
+    tag = "v${version}";
+    hash = "sha256-J4xQY2W5upJQ3hrA2hjkw8voLpxNPpekNwmyMKKAVAo=";
+  };
+
+  build-system = [ poetry-core ];
+  dependencies = [ requests ];
+
+  nativeCheckInputs = [
+    pproxy
+    pytest-socket
+    pysocks
+    trustme
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "requests_hardened" ];
+
+  meta = {
+    description = "Library that adds hardened behavior to python requests";
+    homepage = "https://github.com/saleor/requests-hardened";
+    license = lib.licenses.bsd3;
+    maintainers = [ lib.maintainers.ryand56 ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -16488,6 +16488,8 @@ self: super: with self; {
 
   requests-gssapi = callPackage ../development/python-modules/requests-gssapi { };
 
+  requests-hardened = callPackage ../development/python-modules/requests-hardened { };
+
   requests-hawk = callPackage ../development/python-modules/requests-hawk { };
 
   requests-http-message-signatures =

